rrmngmnt/nmcli.py

```diff
@@ -0,0 +1,737 @@
+# -*- coding: utf-8 -*-
+"""
+This module introduces support for the nmcli command which is provided by
+NetworkManager.
+"""
+import logging
+import shlex
+
+import netaddr
+
+from rrmngmnt.errors import CommandExecutionFailure
+from rrmngmnt.service import Service
+
+IPV4_STATIC = "ipv4.addresses {address} ipv4.gateway {gateway}"
+IPV6_STATIC = "ipv6.addresses {address} ipv6.gateway {gateway}"
+
+IPV4_METHOD = "ipv4.method {method}"
+IPV6_METHOD = "ipv6.method {method}"
+
+COMMON_OPTIONS = (
+    "type {type} "
+    "con-name {con_name} "
+    "ifname {ifname} "
+    "autoconnect {auto_connect} "
+    "save {save}"
+)
+
+IP_MANUAL_METHOD = "manual"
+
+DEFAULT_BOND_MODE = "active-backup"
+
+DEFAULT_MIIMON = 100
+
+ERROR_MSG_FORMAT = "command -> {command}\nRC -> {rc}\nOUT -> {out}\nERROR -> {err}"  # noqa: E501
+
+NMCLI_COMMAND = "nmcli {options} {object} {command}"
+NMCLI_CONNECTION_DELETE = "nmcli connection delete {id}"
+NMCLI_CONNECTION_ADD = "nmcli connection add"
+
+logger = logging.getLogger(__name__)
```

Comment:
avoid defining new loggers it will mess up with logs filters.
please use `Service.logger` instead, directly from your class since it is inhering from service.